core/assets/bundles/bundle_nl.properties

```diff
@@ -118,91 +118,91 @@ players = {0} Spelers online
 players.single = {0} Speler online
 server.closing = [accent]Server aan het sluiten...
 server.kicked.kick = Je bent verwijderd van deze sessie.
-server.kicked.whitelist = You are not whitelisted here.
+server.kicked.whitelist = Je bent hier niet ge-whitelist.
 server.kicked.serverClose = Server afgesloten...
-server.kicked.vote = Je bent ge vote-kicked. Tot ziens.
+server.kicked.vote = Je bent ge-vote-kicked. Tot ziens.
 server.kicked.clientOutdated = Verouderde versie! Update jouw spel!
 server.kicked.serverOutdated = Verouderde server! Vraag de host om te upgraden!
 server.kicked.banned = Je bent verbannen van deze server.
 server.kicked.typeMismatch = Deze server is niet compitabel met jouw bouwtype.
-server.kicked.playerLimit = This server is full. Wait for an empty slot.
+server.kicked.playerLimit = Deze server zit vol. Wacht voor een lege plaats.
 server.kicked.recentKick = Je bent reeds verwijderd.\nWacht voordat je opnieuw verbindt.
 server.kicked.nameInUse = Er is al iemand met die naam\nop deze server.
 server.kicked.nameEmpty = Je gekozen naam is niet geldig.
 server.kicked.idInUse = Je bent al verbonden met deze server! Verbinden met 2 accounts is verboden.
 server.kicked.customClient = Deze server ondersteunt geen aangepaste spellen . Download de officiele versie.
 server.kicked.gameover = Spel afgelopen
 server.versions = Jouw versie:[accent] {0}[]\nServer versie:[accent] {1}[]
-host.info = De [accent]host[] knop hosts `een server op port [scarlet]6567[]. \nIedereen op hetzelfde [LIGHT_GRAY]wifi or locaal netwerk[] zou jouw server in hun serverlijst moeten zien.\n\nAls je wilt dan vrienden vanaf overal kunnen meedoen via IP, [accent]port forwarding[] is nodig.\n\n[LIGHT_GRAY]Note: IAls iemand moeilijkheden heeft met het meedoen aan jouw spel, kijk of je Mindustry in je firewall instellingen toegang hebt gegeven to jouw locaal netwerk.
-join.info = Hier kan je een [accent]server IP[] invoeren om te verbinden, of om[accent]locale netwerken[] te vinden.\nBeide LAN en WAN multiplayer is ondersteund.\n\n[LIGHT_GRAY]Note: Er is geen automatische globale serverlijst; Als je met iemands IP wil verbinden, Zou je moeten vragen om hun IP.
-hostserver = Host Game
-invitefriends = Invite Friends
-hostserver.mobile = Host\nGame
+host.info = De [accent]host[] knop hosts `een server op port [scarlet]6567[]. \nIedereen op dezelfde [LIGHT_GRAY]wifi of locale netwerk[] zou jouw server in hun serverlijst moeten zien.\n\nAls je wilt dat vrienden vanaf overal kunnen meedoen via IP, [accent]port forwarding[] is nodig.\n\n[LIGHT_GRAY]Note: Als iemand moeilijkheden heeft met het meedoen aan jouw spel, kijk of je Mindustry in je firewall instellingen toegang hebt gegeven to jouw locale netwerk.
```

Comment:
"locale" -> "lokale"
Various other grammatical errors.
"Als je wilt dat vrienden vanaf overal kunnen meedoen via IP, [accent]port forwarding[] is nodig." -> ".. via IP is [accent]port forwarding[] nodig."

Reply:
Sorry, I must've misread "locale".

The second error I wasn't sure about, but I think you're right about it